packages/react-native-web/src/exports/Text/index.js

```diff
@@ -116,7 +116,7 @@ const Text = forwardRef<TextProps, *>((props, forwardedRef) => {
   const style = [
     props.style,
     numberOfLines != null && numberOfLines > 1 && { WebkitLineClamp: numberOfLines },
-    selectable === false && styles.notSelectable,
+    selectable === true ? styles.selectable : styles.notSelectable,
```

Comment:
This is not right as it makes text non-selectable by default (this is why the snapshots are failing). If `selectable` is `false`, it should set `userSelect:none`. If it is `true` it should set `userSelect:text`. If undefined/null do nothing

Reply:
Changed!